app/views/dashboard/_navigation.html.erb

```diff
@@ -3,8 +3,14 @@
     <%= component.nav_item(path: dashboard_path) do %>
       Overview
     <% end %>
-    <%= component.nav_item(path: lead_providers_your_schools_path) do %>
-      Schools
+    <% if FeatureFlag.active?(:multiple_cohorts) %>
```

Comment:
Do you still need this conditional? It should always point to the right place.

Reply:
We can drop the conditional, but keep the param in the URL as we want the cohort 2022 schools to be the default view when we flip on multiple cohorts.